app-backend/db/src/main/scala/ProjectDao.scala

```diff
@@ -276,6 +278,18 @@ object ProjectDao
     }
   }
 
+  def getFootprint(projectId: UUID): ConnectionIO[Option[Projected[Geometry]]] =
+    for {
+      projectO <- ProjectDao.getProjectById(projectId)
+      footprint <- projectO traverse { project =>
+        ProjectLayerScenesDao.getUnionedGeomFootprint(project.defaultLayerId)
+      }
+    } yield {
+      footprint.flatten map { fp =>
```

Comment:
This looks weird and sounds like it should be flatMap-able, but it's not really -- I have an `Option[Option[...]]` from the `traverse`